pyproject.toml

```
@@ -30,6 +30,7 @@ dependencies = [
     "packaging>=21.0",
     "tqdm>=4.29.0",
     "joblib>=1.3.0",
+    "numpy<2.0.0"
```

```suggestion
    "numpy<2.0.0" # some of our dependencies are not yet compatible with numpy 2.0
```

Always better to add a comment that doesn't make us forget why this was placed here.